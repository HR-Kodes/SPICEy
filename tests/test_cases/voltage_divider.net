V1 1 0 5
R1 1 2 100
R2 2 0 200