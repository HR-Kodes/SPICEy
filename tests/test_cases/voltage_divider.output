
Node Voltages:
V1 = 5.000 V (fixed)
V2 = 3.333 V
