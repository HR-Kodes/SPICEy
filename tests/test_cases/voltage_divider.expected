{
  "nodes": {
    "1": 5.0,
    "2": 3.333
  },
  "tolerance": 0.001
}